wordpress-mobile/gutenberg-mobile
# [Tech-debt] move `separatorType` of controls to the PanelBody

Fixes part of #1421 

Gutenberg PR - https://github.com/WordPress/gutenberg/pull/21365

To test:
- Check components that use PanelBody with controls in it. The list is in Gutenberg PR.

PR submission checklist:
